app/src/ProtectedRoute.js

```diff
@@ -0,0 +1,24 @@
+import { Navigate, Outlet } from "react-router-dom";
+import React, { useState } from "react";
+
+const response = function IsloggedIn() {
```

Comment:
```suggestion
const IsloggedIn = function() {
```
We should try to create functions using the same notation all the time. Either use function keyword or use arrow notation.